components/ht16k33_alpha/ht16k33_display.cpp

```diff
@@ -55,7 +55,27 @@ void HT16K33AlphaDisplay::loop() {
   }
 }
 
-float HT16K33AlphaDisplay::get_setup_priority() const { return setup_priority::PROCESSOR; }
+float HT16K33BaseDisplay::get_setup_priority() const { return setup_priority::PROCESSOR; }
+
+void HT16K337SegmentDisplay::display_() {
+  int offset = this->offset_;
+  constexpr uint8_t size = 10;
+  uint8_t buffer[size];
+  memcpy(buffer, this->buffer_ + offset, 4);
+// TODO: implement method to show/hide colon symbol
```

Comment:
I'm unsure how to handle that. The easiest would be to add a "show_colon(bool)" method to the base class, and store that in a variable, but this would be useless for the alphanumeric display that does not have a colon :-(